lib/Object.php

```diff
@@ -104,11 +105,11 @@ public function &__get($k)
                     . "probably as a result of a save(). The attributes currently "
                     . "available on this object are: $attrs";
             error_log($message);
-            return null;
+            return $nullval;
         } else {
             $class = get_class($this);
             error_log("Stripe Notice: Undefined property of $class instance: $k");
-            return null;
+            return $nullval;
```

Comment:
On my side they don't pass, using ```return;``` is strictly equals to ```return null;``` and constants can't be passed by reference.